packages/donor/src/screens/unsubscribe/UnsubscribedScreen.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Donor } from "@zm-blood-components/common";
+import { MainNavigationKeys } from "../../navigation/app/MainNavigationKeys";
+import zmDrop from "../../assets/images/zmDrop.svg";
+import styles from "./UnsubscribedScreen.module.scss";
+
+export interface UnsubscribedScreenProps {
+  isLoggedIn: boolean;
+  user?: Donor;
+}
+
+const UnsubscribedScreen = (props: UnsubscribedScreenProps) => {
+  const { user, isLoggedIn } = props;
+  const profileLink = <Link to={MainNavigationKeys.MyProfile}>הפרופיל</Link>;
+  const GOODBYE = "להתראות !";
+  const EMAIL_REMOVED = `${
+    isLoggedIn ? user?.email : "האימייל שלך"
```

Comment:
Even if the user is logged in, nothing says it is logged in with the same email we just unsubscribed, so we might be lying to the user here.

I think we should just always say "האימייל שלך". What do you think?

Reply:
sure, we can add in the task that will redirect to this page to add the email in the redirection, for now I will remove it